README.md

```diff
@@ -127,11 +141,11 @@ This command will create the migrations:
 
 These commands will then apply the newly created migrations to the database:
 
-`./manage.py migrate rider 0001
+`./manage.py migrate rider 0001`
```

Comment:
This fails to run for me because of a problem with psycopg2. It seems that setup.sh doesn't manager to install psycopg2 and when I try to manually install it via pip, it fails to install.